script/demoScripts/utils/demoScriptHelpers.ts

```
@@ -15,6 +15,7 @@ import {
   http,
   Narrow,
   parseAbi,
+  zeroAddress,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'zeroAddress'. Consider to delete it or rename it.

'zeroAddress' is defined here:


(lint/suspicious/noRedeclare)